Lib/pickle.py

```diff
@@ -1705,8 +1707,8 @@ def load_build(self):
         stack = self.stack
         state = stack.pop()
         inst = stack[-1]
-        setstate = getattr(inst, "__setstate__", None)
-        if setstate is not None:
+        setstate = getattr(inst, "__setstate__", _NoValue)
+        if setstate is not _NoValue:
```

Comment:
This change is not tested.

Reply:
I added a test for this. It might be the changes for `Pickler.reducer_override` are not tested either, I will check this later.

Reply:
@erlend-aasland All changes from this PR should be covered by tests now.